crates/libs/bindgen/src/extensions/in_addr.rs

```diff
@@ -0,0 +1,26 @@
+use super::*;
+
+pub fn gen() -> TokenStream {
+    quote! {
+        impl ::core::convert::From<::std::net::Ipv4Addr> for IN_ADDR {
+            fn from(addr: ::std::net::Ipv4Addr) -> Self {
+                Self { S_un: IN_ADDR_0 { S_addr: u32::from(addr) } }
+            }
+        }
+        impl ::core::convert::From<&::std::net::Ipv4Addr> for IN_ADDR {
+            fn from(addr: &::std::net::Ipv4Addr) -> Self {
+                Self { S_un: IN_ADDR_0 { S_addr: u32::from(*addr) } }
+            }
+        }
+        impl ::core::convert::Into<::std::net::Ipv4Addr> for &IN_ADDR {
```

Comment:
```suggestion
        impl ::core::convert::From<&IN_ADDR> for ::std::net::Ipv4Addr {
```

As said by kennykerr, it's discouraged to implement Into directly, so best to swap the type types and implement From instead

Reply:
Done.